src/components/widgets/Form.js

```diff
@@ -0,0 +1,315 @@
+import React, { Component } from 'react'
+import { RadioGroup, Radio } from 'react-radio-group'
+import Button from '../Button.js'
+import "../../styles/form.css"
+
+export default class Form extends Component {
+  constructor( props ) {
+    super( props )
+    this.state = {}
+    this.setFieldState = this.setFieldState.bind( this )
```

Comment:
What happens if this isn't here? I thought the ES6 class already bound methods to this?